app/assets/stylesheets/ideas.scss

```diff
@@ -1,3 +1,10 @@
-// Place all the styles related to the Ideas controller here.
-// They will automatically be included in application.css.
-// You can use Sass (SCSS) here: http://sass-lang.com/
+#deleted {
+  text-align: center;
+  background: #ffe;
+  width:100%;
+  margin-left:auto;
+  margin-right:auto;
+  border:1px solid gray;
```

Comment:
Color `gray` should be written in hexadecimal form as `#808080`<br>Color literals like `gray` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Colon after property should be followed by one space